pkg/process/util/util.go

```diff
@@ -120,9 +120,9 @@ func GetPlatform() (string, error) {
 }
 
 // IsDebugfsMounted would test the existence of file /sys/kernel/debug/tracing/kprobe_events to determine if debugfs is mounted or not
-func IsDebugfsMounted() bool {
+func IsDebugfsMounted() error {
```

Comment:
Not a fan of this - @sfluor just changed this somewhere else.

We should have it return `(bool, message)`